src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs

```diff
@@ -488,4 +488,68 @@ public async Task<IActionResult> RejectDisputeUpdateRequestAsync(long updateStat
         await _disputeService.RejectDisputeUpdateRequestAsync(updateStatusId, cancellationToken);
         return Ok();
     }
+
+    /// <summary>
+    /// Returns all Disputes that have pending update requests from the Oracle Data API
+    /// </summary>
+    /// <param name="cancellationToken"></param>
+    /// <response code="200">The Disputes were found.</response>
+    /// <response code="401">Request lacks valid authentication credentials.</response>
+    /// <response code="403">Forbidden, requires dispute:read permission.</response>
+    /// <response code="500">There was a server error that prevented the search from completing successfully or no data found.</response>
+    /// <returns>A collection of Dispute records</returns>
+    [HttpGet("disputesWithUpdateRequests")]
+    [ProducesResponseType(typeof(IList<Dispute>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [KeycloakAuthorize(Resources.Dispute, Scopes.Read)]
+    public async Task<IActionResult> GetDisputesWithPendingUpdateRequestsAsync(CancellationToken cancellationToken)

```

Comment:
It would be nice to see XUnit tests for each of these endpoints.

Reply:
added